app/api/v1/websocket/feedme_websocket.py

```
@@ -0,0 +1,493 @@
+                except asyncio.TimeoutError:
+                    # Send heartbeat if no message received
+                    await realtime_manager.send_heartbeat(room_id)
+                
+        except WebSocketDisconnect:
+            logger.info(f"Processing WebSocket disconnected: user={user_id}, conversation={conversation_id}")
+        
+    except Exception as e:
+        logger.error(f"Error in processing WebSocket: {e}")
+        try:
+            await websocket.close(code=1011, reason="Internal server error")
+        except:
```

Flake8 (7.2.0)

[error] do not use bare 'except'

(E722)